docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:8.0-cudnn6-devel
+FROM nvidia/cuda:9.0-cudnn7-devel
```

Comment:
Why do you use `9.0` ? 
`9.2` looks available.
https://hub.docker.com/r/nvidia/cuda/

Reply:
For compatibility. If you want to know in details, please refer to below.
https://github.com/tensorflow/tensorflow/issues/15604

Reply:
ok.